### PR TITLE
Clarifying relations between User, Account and Actor at the domain model level

### PR DESCRIPTION
**Pull request by yvolk**
Clarifying relations between User, Account and Actor at the domain model level (i.e. in the Overview section) as per our discussion at #260

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -155,8 +156,9 @@
       </p>
 
       <p>
-        In ActivityPub, accounts on a server (including those of real-world users, bots,
-        and other automated processes) are represented by "<a href="#actors">actors</a>". 
+        In ActivityPub, a user is represented by "<a href="#actors">actors</a>"
+        via the user's accounts on servers. In this version of ActivityPub
```
Comment (cwebber) [inline index.html]: I'm happy with everything in this change, except could we tweak this line?  "In this version of ActivityPub" seems to imply that we'll push out a new ActivityPub revision where the actor isn't the base component, which I think (even if we have more domain modeling to do) that's going to remain the case.

How about "in this specification"?

**Comment by nightpool** [comment]
Why include the line, "an account corresponds to exactly one actor"? That's needlessly and incredibly restrictive, preventing any kind of Tumblr-like "side blog" setup or any kind of company group actor, like tweetdecks "authorized account" feature.

Accounts are completely out of scope for activitypub anyway. I'm fine including more examples of actors but any language beyond that should not be introduced.

**Comment by cwebber** [comment]
@nightpool It's a good point... maybe making that particular change adds confusion... maybe the existing text for that paragraph is already good enough.  (We've already iterated on it several times over the course of #260 so maybe it's improved enough already.)  The other two diff chunks are good though I think?

**Comment by nightpool** [comment]
changes look fine to me other then the "users are represented via accounts, with exactly one actor per account" one.

File: index.html (hunk 1 above)
Comment (yvolk) [reply index.html]: I deleted "In this version of ActivityPub" as you suggested. Further discussion - in issue #260

**Comment by jaywink** [comment]
I agree, "one actor per account" is not a beneficial or necessary change, IMHO. As said multiple times, accounts are out of scope of AP. An Actor *can* be an account, but for AP that doesn't care if it is not. An account is an implementation detail, for example Tweetdeck and Facebook allow multiple accounts to act on behalf of an Actor (a Page).

**Comment by yvolk** [comment]
@jaywink in #260 we discussed for a month notions of a user, account and actor and their relations between each other.
1. My today's comments, starting from https://github.com/w3c/activitypub/issues/260#issuecomment-341626964 will help you understand, why I think that we need to conceptually describe these relations in the Overview.
2. Regarding your example with Tweetdeck: we mentioned it in the thread, please see my reply here: https://github.com/w3c/activitypub/issues/260#issuecomment-340250958
3. "Facebook allow multiple accounts to act on behalf of an Actor (a Page)." Please see my comment on "Why one-to-one relation of Actor to Account is an edge case" here: https://github.com/w3c/activitypub/issues/260#issuecomment-339414883

Please reply to the main thread.

**Comment by nightpool** [comment]
the conversation in #260 is hard to follow, pointlessly insists on using confusing terminology, and nearly 10,000 words. I don't need to engage with the whole thing to oppose these object-level changes, and any norms that require me to are bad discourse norms.

**Comment by yvolk** [comment]
@nightpool We started a month ago with "users _are_ actors here"...
We are discussing "domain level" description of ActivityPub, its relation to the real world, system landscape, and not "object-level changes". Please try to look from that point.

**Comment by cwebber** [comment]
I think the most recent changes address @nightpool's concerns, so I'm pretty happy with this.  Merging!
